docs/application/web/tutorials/sec-privileges.md

```diff
@@ -19,214 +19,98 @@ Since Tizen platform 3.0, some privileges are categorized as privacy-related and
-| `http://tizen.org/privilege/mediacapture` | public | Camera and Microphone | 2.2.1 | The application can manipulate streams from cameras and microphones using the [getUserMedia](https://developer.tizen.org/dev-guide/latest/org.tizen.web.apireference/html/w3c_api/w3c_api_m.html#getusermedia) API.<br> **Privilege behavior:**<br> In the local domain, if this privilege is defined, permission is granted. Otherwise, execution is blocked.In the remote domain, if this privilege is defined, pop-up user prompt is used. Otherwise, execution is blocked. |
-| `http://tizen.org/privilege/unlimitedstorage` | public | -                     | 2.2.1 | The application can use the storage with unlimited size with the [File API: Directories and System](https://developer.tizen.org/dev-guide/latest/org.tizen.web.apireference/html/w3c_api/w3c_api_m.html#directory), [File API: Writer](https://developer.tizen.org/dev-guide/latest/org.tizen.web.apireference/html/w3c_api/w3c_api_m.html#writer), [Indexed Database](https://developer.tizen.org/dev-guide/latest/org.tizen.web.apireference/html/w3c_api/w3c_api_m.html#database), and [Web SQL Database](https://developer.tizen.org/dev-guide/latest/org.tizen.web.apireference/html/w3c_api/w3c_api_m.html#sql) APIs.<br>**Privilege behavior:**<br>In the local domain, if this privilege is defined, permission is granted. Otherwise, pop-up user prompt is used.In the remote domain, pop-up user prompt is used. |
-| `http://tizen.org/privilege/notification` | public | -                     | 2.2.1 | The application can display simple notifications using the [Web Notifications](https://developer.tizen.org/dev-guide/latest/org.tizen.web.apireference/html/w3c_api/w3c_api_m.html#webnoti) API.<br>**Privilege behavior:**<br>In the local domain, if this privilege is defined, permission is granted. Otherwise, pop-up user prompt is used.In the remote domain, pop-up user prompt is used. |
-| `http://tizen.org/privilege/location`    | public | Location              | 2.2.1 | The application can access geographic locations using the [Geolocation](https://developer.tizen.org/dev-guide/latest/org.tizen.web.apireference/html/w3c_api/w3c_api_m.html#geo) API.<br>**Privilege behavior:**<br>In the local domain, if this privilege is defined, permission is granted. Otherwise, execution is blocked.In the remote domain, if this privilege is defined, pop-up user prompt is used. Otherwise, execution is blocked. |
-
-**Table: Mobile Web Supplementary API privileges**
-
-| Privilege                               | Level  | Since | Description                              |
+<a name="API"></a>
```

Comment:
line 6: if not used properly
line 12: gives an option to the user to switch them **on** and **off**.
switched it **on**.

Suggestion:
line  12 can come under the Note

Check the hyperlink in line 18

Reply:
updated line 6, 12, and 18.